### DISCUSSION TITLE
What is AM5+?

### DISCUSSION THREAD

**Discussion by ghost** [platomav/MCExtractor#68] [Q&A] [answered]
What is AM5+?
https://github.com/platomav/MCExtractor/releases/tag/v1.101.0-r293

Where did this information come from?
Do you have any more information on what AM5+ is?

**Comment by platomav** [comment, marked as answer]
There is no AM5+, AFAIK. This is a misunderstanding.

I've seen some articles mentioning a rumored "AM5+" socket based on the changelog of MCE. However, the "+" symbol there simply referred to "AM5 and newer/future" platforms, not an actual socket change by AMD.

I forgot about the old AM3+ when writing this. I will keep that in mind from now on to avoid the "+" symbol as a shortcut for AMD. 🙄 In fact, I had used the "AM5+" in the same manner back in September 2023 (https://github.com/platomav/MCExtractor/releases/tag/v1.92.1-r275). It does not mean anything.

**Comment by ghost** [comment]
Thank you :)
